modules/core/src/main/java/org/apache/ignite/internal/processors/cache/GridCacheProxyImpl.java

```diff
@@ -294,7 +294,7 @@ public GridCacheProxyImpl(
                         false,
                         null,
                         false,
-                        false,
+                        null,
```

Comment:
Here and below, let's replace it with `CacheOperationContext.keepBinary()`

Reply:
Let's refactor this at another issue if necessary.